ext/session/session.c

```diff
@@ -1724,6 +1730,12 @@ static PHP_FUNCTION(session_set_cookie_params)
 		zend_string_release(ini_name);
 	}
 
+    if (argc > 5) {
+        ini_name = zend_string_init("session.cookie_samesite", sizeof("session.cookie_samesite") - 1, 0);
+        zend_alter_ini_entry(ini_name, samesite, PHP_INI_USER, PHP_INI_STAGE_RUNTIME);
```

Comment:
In fact... ugh. Are these parameters really all just wrappers for ini_set()?  Let's stop that madness and just let people use ini_set().  This is a silly function.

Reply:
Too bad that we decided on 2017-10-10 to alternatively allow to pass an option array to this function. Anyhow, I think `session_set_cookie_params()` and maybe `session_get_cookie_params()` might be worthwhile additions to https://wiki.php.net/rfc/deprecations_php_7_4.

Reply:
@cmb69 feel free to add anything you please to the RFC, its an open WIP